js/src/components/modals/PostSettingsModal.js

```diff
@@ -0,0 +1,58 @@
+import { Button } from "@yoast/components/src/button/Button";
+import styled from "styled-components";
+import { __, sprintf } from "@wordpress/i18n";
+import { useState, useCallback } from "@wordpress/element";
+import Modal from "./Modal";
+
+/**
+ * Container for the button.
+ *
+ * Sets the flex-direction to column to make the button full-width.
+ */
+const Container = styled.div`
+	padding: 16px;
+	display: flex;
+	flex-direction: column;
+	border-bottom: 1px solid rgba( 0,0,0,0.2 );
+`;
+
+/**
+ * Returns a button in a div that can be used to open the modal.
+ *
+ * Warning: contains styling that is specific for the Sidebar.
+ *
+ * @returns {*} A button wrapped in a div.
+ */
+const PostSettingsModal = () => {
+	const [ isOpen, changeIsOpen ] = useState( false );
+
+	const closeModal = useCallback( () => changeIsOpen( false ), [] );
+	const openModal = useCallback( () => changeIsOpen( true ), [] );
+
+	return (
+		<Container
+			className="yoast"
```

Comment:
Seems strange that this is needed to make our button CSS apply. Possibly the SidebarSlot should have this in the future. I'm not too sure though, so I'm fine to leave this now.

Reply:
Yeah, this has to do with the specificity of the CSS selectors. Apparently, it was not enough to give them "unique" yoast classes. I like your idea of moving that to the SidebarSlot in the future though. But for now, this will do indeed :)